### PR TITLE
875229 UG documentation for E-Signature

### PR DESCRIPTION
**Pull request by VijayadharshiniMathiyalagan**

### CODE REVIEW

**Review by Deepak1799 — APPROVED** [review]
Approved

File: MAUI/PDF-Viewer/Signature.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: Electronic Signature in .NET MAUI PDF Viewer control | Syncfusion
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:3]: title: Electronic Signature in .NET MAUI PDF Viewer Control | Syncfusion

File: MAUI/PDF-Viewer/Signature.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+description: Learn here all about Electronic Signature in Syncfusion .NET MAUI PDF Viewer (SfPdfViewer) control and its types.
+platform: MAUI
+control: SfPdfViewer
+documentation: ug
+---
+
+# Electronic Signature in .NET MAUI PDF Viewer (SfPdfViewer)
+
+The electronic signature feature of [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to add, remove signature in the PDF document. This section will go through the various types and functions available in PDF Viewer for working with signature.
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:12]: The electronic signature feature of [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to add, remove signature in the PDF document. This section will go through the various types and functions available in the PDF Viewer for working with signature.

File: MAUI/PDF-Viewer/Signature.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+1.	Handwritten signature.
+2.	Text signature.
+3.  Image signature.
+
+## Add signature by UI interaction
+
+This section will go through how to add signature by UI interaction to a PDF document.
+
+You can add signature to a PDF document with UI interaction using signature dialog. The following steps explains how to add signature on a PDF.
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:26]: You can add signature to a PDF document with UI interaction using the signature dialog. The following steps explains how to add signature on a PDF.

File: MAUI/PDF-Viewer/Signature.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+	1. Set the AnnotationMode property of the SfPdfViewer to `Signature`. It activates the signature mode on the control and signature dialog box will be open.
+	2. create a signature in any one of the types
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:29]: 2. Create a signature in any one of the types.

File: MAUI/PDF-Viewer/Signature.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+	3. You can customize the color of the signature for handwritten and text signature types
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:30]: 3. You can customize the color of the signature for handwritten and text signature types.

File: MAUI/PDF-Viewer/Signature.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+// Enable or activate the signature mode.
+void EnableSignatureMode()
+{
+    // Set the annotation mode to signature using the `SfPdfViewer` instance.
+    PdfViewer.AnnotationMode = AnnotationMode.Signature;
+}
+{% endhighlight %}
+{% endtabs %}
+
+Similarly, refer to following code to disable the signature mode.
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:48]: Similarly, refer to the following code to disable the signature mode.

File: MAUI/PDF-Viewer/Signature.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+    PdfViewer.AnnotationMode = AnnotationMode.None;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Add signature programmatically
+
+This section will go through how to add signature programmatically to a PDF document.
+
+Note: Currently, there is no support for add a text signature programmatically.
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:65]: Note: Currently, there is no support for adding a text signature programmatically.

File: MAUI/PDF-Viewer/Signature.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+### Add handwritten signature 
+
+To add a handwritten signature, You can create and add an ink annotation to a PDF document programmatically using the AddAnnotation method of the SfPdfViewer. Additionally in the case of signature you should set the `IsSignature` API of the ink annotation true. The following example explains how to create an ink annotation and add it as a signature to the first page of a PDF document.
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:69]: To add a handwritten signature, You can create and add an ink annotation to a PDF document programmatically using the AddAnnotation method of the SfPdfViewer. Additionally, in the case of signature, you should set the `IsSignature` API of the ink annotation to true. The following example explains how to create an ink annotation and add it as a signature to the first page of a PDF document.

File: MAUI/PDF-Viewer/Signature.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+{% tabs %}
+{% highlight c# %}
+int pageNumber = 1;
+    
+// Provide the points collection to draw a stroke. Here a single stroke is created.
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:75]: // Provide the points collection to draw a stroke. Here, a single stroke is created.

File: MAUI/PDF-Viewer/Signature.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+   new List<float> { 40, 300, 60, 100, 40, 50, 40, 300 }
+};
+
+// Create an ink annotation.
+InkAnnotation annotation = new InkAnnotation(pointsCollection, pageNumber);
+
+// Set `IsSignature` as true
+annotation.IsSignature = true;
+
+// Add the ink annotation to the PDF page as signature
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:87]: // Add the ink annotation to the PDF page as signature.

File: MAUI/PDF-Viewer/Signature.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+PdfViewer.AddAnnotation(annotation);
+{% endhighlight %}
+{% endtabs %}
+
+### Add image signature
+
+To add an image signature You can create and add a custom stamp annotation to a PDF document programmatically using the AddAnnotation method of the SfPdfViewer. Additionally in the case of signature you should set the `IsSignature` API of the stamp annotation true. The following example explains how to create a custom stamp annotation and add it as a signature to the first page of a PDF document
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:94]: To add an image signature, you can create and add a custom stamp annotation to a PDF document programmatically using the AddAnnotation method of the SfPdfViewer. Additionally, in the case of signature, you should set the `IsSignature` API of the stamp annotation to true. The following example explains how to create a custom stamp annotation and add it as a signature to the first page of a PDF document.

File: MAUI/PDF-Viewer/Signature.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+
+{% tabs %}
+{% highlight c# %}
+int pageNumber = 1;
+
+// Define the position and size for the stamp to be placed in the PDF page.
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:100]: // Define the position and size of the stamp to be placed in the PDF page.

File: MAUI/PDF-Viewer/Signature.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+// Create image stream from the image to be used as stamp.
+Stream imageStream = this.GetType().Assembly.GetManifestResourceStream("Annotations.Assets." + "Logo.png");
+
+// Create a custom stamp annotation using the image steeam.
+StampAnnotation customStamp = new StampAnnotation(imageStream,pageNumber,bounds);
+
+// Set `IsSignature` as true
+customStamp.IsSignature = true;
+
+// Add the stamp annotation to the PDF page as signature
```
Comment (M-A32) [inline MAUI/PDF-Viewer/Signature.md:112]: // Add the stamp annotation to the PDF page as signature.

**Review by M-A32 — COMMENTED** [review]
I have completed the review.

**Pull request merged**